test/controllers/user.test.js

```diff
@@ -0,0 +1,36 @@
+const supertest = require('supertest');
+const models = require('../../app/models/index');
+const app = require('../../app');
+
+const request = supertest(app);
+
+describe('usersController.signUp', () => {
+  it('Creates a user', () => {
```

Comment:
This one is fine, but you should test for border cases too. I'll let you think about what border cases I'm refering to, there are 3 of them